microsetta_interface/implementation.py

```diff
@@ -929,7 +936,12 @@ def get_source(*, account_id=None, source_id=None):
             needs_assignment = True
         else:
             dt = datetime.fromisoformat(sample['sample_datetime'])
-            sample['sample_datetime'] = dt.strftime("%b-%d-%Y %-I:%M %p")
+            # sample['sample_datetime'] = dt.strftime("%b-%d-%Y %-I:%M %p")
+            # rebase=True - show in user's locale, rebase=False, UTC (I think?)
```

Comment:
I believe this is `BABEL_DEFAULT_TIMEZONE` which is set to `PT` right now

Reply:
rebase=False will show in whatever the BABEL_DEFAULT_TIMEZONE is yeah.  If you set it to PT somewhere that's fine.

Reply:
kk